sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/AzureSdkLoggingTests.cs

```diff
@@ -0,0 +1,182 @@
+
+            builder.Services.TryAddSingleton<Microsoft.Extensions.Azure.AzureEventSourceLogForwarder>();
+            using var app = builder.Build();
+
+            Microsoft.Extensions.Azure.AzureEventSourceLogForwarder publicLogForwarder =
+                app.Services.GetRequiredService<Microsoft.Extensions.Azure.AzureEventSourceLogForwarder>();
+
+            Assert.NotNull(publicLogForwarder);
+            publicLogForwarder.Start();
+
+            await app.StartAsync();
+
+            using TestEventSource source = new TestEventSource();
+            Assert.True(source.IsEnabled());
+            source.LogMessage("hello", eventLevel);
+
+            WaitForRequest(transport);
+            if (expectedMessage != null)
+            {
+                Assert.Single(transport.Requests);
+                await AssertContentContains(transport.Requests.Single(), expectedMessage, eventLevel);
+            }
+            else
+            {
+                Assert.False(transport.Requests.Any());
+            }
+        }
+
+        private void WaitForRequest(MockTransport transport)
+        {
+            SpinWait.SpinUntil(
+                condition: () =>
+                {
+                    Thread.Sleep(10);
+                    return transport.Requests.Count > 0;
+                },
+                timeout: TimeSpan.FromSeconds(10));
+        }
+
+        private static async Task AssertContentContains(MockRequest request, string expectedMessage, LogLevel expectedLevel)
+        {
+            using var contentStream = new MemoryStream();
+            await request.Content.WriteToAsync(contentStream, default);
+            contentStream.Position = 0;
+            var content = BinaryData.FromStream(contentStream).ToString();
+            var jsonMessage = $"\"message\":\"{expectedMessage}\"";
+            var jsonLevel = $"\"severityLevel\":\"{expectedLevel}\"";
+            Assert.Contains(jsonMessage, content);
+            Assert.Contains(jsonLevel, content);
+
+            // also check that message appears just once
+            Assert.Equal(content.IndexOf(jsonMessage), content.LastIndexOf(jsonMessage));
+        }
+
+        private static void SetUpOTelAndLogging(WebApplicationBuilder builder, MockTransport transport, LogLevel enableLevel)
+        {
+            builder.Logging.ClearProviders();
+            builder.Logging.AddFilter((name, level) =>
+            {
+                if (name != null && name.StartsWith("Azure"))
+                {
+                    return level >= enableLevel;
+                }
+                return false;
+            });
+
+            builder.Services.AddOpenTelemetry().UseAzureMonitor(config =>
+            {
+                config.Transport = transport;
+                config.ConnectionString = $"InstrumentationKey={Guid.NewGuid()}";
+                config.EnableLiveMetrics = false;
+                config.Diagnostics.IsDistributedTracingEnabled = false;
```

Comment:
Out of curiosity, how do these config.Diagnostics.* property changes affect the log forwarder?

Reply:
it doesn't, it affects additional telemetry captured by the test.

The `MockTransport.Requests` that I use to validate collected telemetry contains calls to quickpulse endpoint.
Also OTel distro is yet another Azure SDK that emits logs and traces - we should disable those by default BTW.